### PR TITLE
Add css and playground

### PR DESCRIPTION
**Pull request by bidoubiwa**
In this PR:

- Moved all examples to `tests/env` folder as they were not really examples.
- Add `basic_search.css` template for ease of integration
- Add HTML playground example
- refactor CI commands

### CODE REVIEW

File: playgrounds/html/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+[![Nobel prices demo](asset/stardew.gif)](https://nobel.meilisearch.com)
+
+## Project setup
+
+```bash
+$ yarn install
+```
+
+### Compiles and hot-reloads for development
```
Comment (curquiza) [inline playgrounds/html/README.md:22]:
```suggestion
### Compile and hot-reload for development
```

File: playgrounds/html/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+```
+yarn serve
+```
+
+### Compiles and minifies for production
```
Comment (curquiza) [inline playgrounds/html/README.md:27]:
```suggestion
### Compile and minify for production
```

File: playgrounds/html/README.md (hunk 1 above)
Comment (curquiza) [reply playgrounds/html/README.md:22]: To make it consistent with the last title.

File: playgrounds/html/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+```
+yarn build
+```
+
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
+
```
Comment (curquiza) [inline playgrounds/html/README.md:34]:
```suggestion
### Customize configuration

See [Configuration Reference](https://cli.vuejs.org/config/).
```

**Review by curquiza — CHANGES_REQUESTED** [review]
Small changes, but everything looks good to me 😁

**Review by curquiza — APPROVED** [review]
perfecto

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**